hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -66,9 +73,41 @@ def set_next_pc(self, next_pc: int) -> None:
         assert(self.is_pc_valid(next_pc))
         self._pc_next_override = next_pc
 
-    def set_rnd_data(self, rnd_data: int) -> None:
+    def step_edn(self, rnd_data: int) -> None:
+        # Take the new data
```

Comment:
What happens if `step_edn` is called when `self.rnd_cdc_pending`? I don't think this should happen in the design, so maybe add a check here and spit out an error?